### PR TITLE
Restaurants page- Additions 

### PR DESCRIPTION
**Pull request by Lordnickim**
Added Filter Feature 
Updated Restaurant Page to include UserId for Orders
Updated Overflow styling for now
Included get requests to server for data hooked up

### CODE REVIEW

**Review by ZedWang1874 — APPROVED** [review]
It is all good. Check.

File: src/client/components/RestaurantsPage.jsx [hunk 1]
```diff
@@ -1,15 +1,137 @@
...
+border-bottom: 1px solid #FF6C36;
+`;
 
-  return ()
+const OrangeCheckBox = styled(Form.Checkbox)`
+
+`;
+
+export default function RestaurantPage() {
+  const userId = useAppSelector((state)=>state.loginDetails.userId)
```
Comment (ZedWang1874) [inline src/client/components/RestaurantsPage.jsx:30]: where this loginDetails comes from?

**Review by ZedWang1874 — APPROVED** [review]
All is good.

File: src/client/components/RestaurantsPage.jsx (hunk 1 above)
Comment (Lordnickim) [reply src/client/components/RestaurantsPage.jsx:30]: From loginDetailsReducer

**Pull request closed without merging**